### ISSUE TITLE
TB Launcher

### ISSUE THREAD

**Issue by TBog** [HunterArmstrong/DennisHunterArmstrong#1]
https://play.google.com/store/apps/details?id=rocks.tbog.tblauncher

What do you think?

**Comment by HunterArmstrong** [comment]
@TBog Are you asking for my opinion?

**Comment by TBog** [comment]
@HunterArmstrong yes I am. I stripped what I don't use from KISS and implemented widgets and icon packs from scratch.

**Comment by HunterArmstrong** [comment]
@TBog I'll look at it later. I'm kind of in a tough spot rn.

**Comment by HunterArmstrong** [comment]
Okay.

@TBog 

A *small* issue. My phone wants to immediately go to the Samsung UI Home page whenever I exclude an app. Why is that happening?

**Comment by TBog** [comment]
It's a crash. I don't use that feature and so haven't tested it after I reworked how the launcher stores app information in the DB

**Comment by HunterArmstrong** [comment]
@TBog 

Okay. I really don't want to sound like a dick. But it seems that you need to test that feature out some more before you include it. Also, there isn't a way to show the apps that were excluded.

**Comment by TBog** [comment]
How about the features that do work?
Widgets, icon packs, adaptive icons, customized colors (the next release will allow for any hex color), gradient or rounded-corners background, customizable memory footprint

**Comment by HunterArmstrong** [comment]
@TBog Everything else works just fine. Just pointing stuff out that needs work. 

Sorry.

How 'Bout this, I'll use this for a couple days. Soon, I will give you the full list of the pros and cons.

**Comment by HunterArmstrong** [comment]
@TBog 

I wrote some notes so far on the TinyBit launcher. Which do you want first; The pros, The cons, Questions I need to ask, or suggestions I want to make?

**Comment by TBog** [comment]
You can leave all of them [here](https://github.com/TBog/TBLauncher/issues)
ar anywhere you prefer.
Start with the questions and suggestions.

**Comment by HunterArmstrong** [comment]
@TBog

Okay. Well, I already left some issues on your GitHub.

First, how are the apps sorted? It just confused me how they weren't sorted alphabetically.

**Comment by TBog** [comment]
The filters don't currently sort the results. I'll look into adding a preference for the order of results when filtering.

**Comment by HunterArmstrong** [comment]
Okay. You do that.

I also think there should be a tutorial for how TB works. Also some tutorials on how to make different styles or customizations.

**Comment by HunterArmstrong** [comment]
@TBog 

I have a question. How often do you plan on updating Tinybit? Are there any issues/pull requests that are a work in progress right now?

**Comment by TBog** [comment]
You can check the commit history on GitHub. I will update when I feel a new version is warranted or a fix is needed. But it seems Google takes a few days to publish.

**Comment by HunterArmstrong** [comment]
Okay. Other than that stuff happening, how was your day so far?

**Comment by TBog** [comment]
Can't work because I don't have a sitter atm.

**Comment by HunterArmstrong** [comment]
Sitter? Like Babysitter?

**Comment by HunterArmstrong** [comment]
Hey, @TBog 

I just saw the commits you made to TB. Good job on fixing the exclude app crash and adding the tag autocomplete function. I can't wait to see both of these things come to action in an update. Keep up the good work!

**Comment by HunterArmstrong** [comment]
I noticed the new commit involving the newest update. Is it being processed by Google play right now?

**Comment by TBog** [comment]
Update v4.5 pending.
The update has been submitted today and is being processed.

**Comment by HunterArmstrong** [comment]
Awesome. I hope everything functions right.🤞🏻🤞🏻

**Comment by HunterArmstrong** [comment]
Hey. Quick question. What does left hand icon on the search bar do?

**Comment by TBog** [comment]
> Hey. Quick question. What does left hand icon on the search bar do?

It restarts all providers, effectively refreshing apps, contacts, shortcuts

**Comment by HunterArmstrong** [comment]
Oookay. But, why does the button have an apps icon instead of a refresh icon?

**Comment by TBog** [comment]
Because that is the launcher icon.
The action that is currently assigned to it is a placeholder. I intend to have that action customizable.

**Comment by HunterArmstrong** [comment]
Yeah. That may be a good idea.

**Comment by HunterArmstrong** [comment]
Will the function be customizable as well? Or just the icon?

**Comment by HunterArmstrong** [comment]
Hey, are you still figuring out how to make the "exclude app" button functional?

**Comment by HunterArmstrong** [comment]
Hey @TBog 

Can I ask what time zone you are in?

**Comment by TBog** [comment]
Eastern European Summer Time
Time zone in Romania (GMT+3)

**Comment by HunterArmstrong** [comment]
Okay. Just wondering.😊

**Comment by HunterArmstrong** [comment]
Hey, I have a question.

Say you have an app that create shortcuts that lead to another separate app. 

If you delete that app, will the shortcuts that were created from it become unpinned?

Or will the shortcuts stay pinned because they don't go to the app that created it?

**Comment by TBog** [comment]
Newer android versions handle the shortcuts outside of the launcher and because of this you can't have invalid shortcuts anymore.
I'll come back with more info when I get to the PC

**Comment by HunterArmstrong** [comment]
Sure thing m8

**Comment by HunterArmstrong** [comment]
Hey, @TBog the app release button on the new update isn't working right now. Is it not done yet? Or is there an error?

**Comment by TBog** [comment]
I am not sure what issues you are having. Is there a button in the app that doesn't work?
